src/main/java/org/springframework/cloud/deployer/spi/kubernetes/ReadinessHttpProbeCreator.java

```diff
@@ -20,25 +20,24 @@
 import org.springframework.util.StringUtils;
 
 /**
- * Creates a Readiness Probe.
+ * Creates an HTTP Readiness Probe.
  *
  * @author Chris Schaefer
  * @author Ilayaperumal Gopinathan
  */
-class ReadinessProbeCreator extends ProbeCreator {
+class ReadinessHttpProbeCreator extends HttpProbeCreator {
+	private final String propertyPrefix;
 
-	private final String probPropertyPrefix;
-
-	public ReadinessProbeCreator(KubernetesDeployerProperties kubernetesDeployerProperties,
-			ContainerConfiguration containerConfiguration) {
+	public ReadinessHttpProbeCreator(KubernetesDeployerProperties kubernetesDeployerProperties,
+									 ContainerConfiguration containerConfiguration) {
 		super(kubernetesDeployerProperties, containerConfiguration);
-		this.probPropertyPrefix = (containerConfiguration.getAppDeploymentRequest() instanceof ScheduleRequest) ?
+		this.propertyPrefix = (containerConfiguration.getAppDeploymentRequest() instanceof ScheduleRequest) ?
 				"spring.cloud.scheduler.kubernetes.readiness" : "spring.cloud.deployer.kubernetes.readiness";
 	}
 
 	@Override
 	public Integer getPort() {
-		String probePortKey = this.probPropertyPrefix + "ProbePort";
+		String probePortKey = this.propertyPrefix + "ProbePort";
```

Comment:
Should we check the port to see if its a numeric like what was done in the ReadinessTcpProbeCreator class?